.maintain/chaostest/LICENSE

```diff
@@ -0,0 +1,674 @@
+                    GNU GENERAL PUBLIC LICENSE
```

Comment:
Does the `chaostest` project need a separate license? Isn't https://github.com/paritytech/substrate/blob/master/LICENSE enough?